sdk/eventhub/Azure.Messaging.EventHubs/src/EventData.cs

```diff
@@ -42,10 +42,17 @@ public class EventData
         /// <seealso cref="BinaryData" />
         /// <seealso cref="EventData.Properties" />
         ///
-        public BinaryData EventBody
+        public BinaryData EventBody => _amqpMessage.GetEventBody();
```

Comment:
public setter is no longer needed (this was only shipped in a beta)

Reply:
I don't mind keeping it - your call 😄

Reply:
cool.   Let's keep them then.